packages/core/admin/admin/src/components/MainNav/tests/NavLink.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@tests/utils';
+
+import { NavLink } from '../NavLink';
+
+describe('NavLink', () => {
```

Comment:
can we make the test more realistic please? so the badge content would probably be an SVG, we need to make sure it renders correctly etc.